include/swift/Basic/Malloc.h

```diff
@@ -44,4 +44,4 @@ inline void AlignedFree(void *p) {
   
 }
 
-#endif
+#endif /* __SWIFT_MALLOC_H__ */
```

Comment:
...unless, from the attached link: "When writing a header file that may be #included by a C source file."

Specifically, I thought it's a better convention to use /* */ comment in the last line of the file, for the unfortunate case when a newline is missing at the end of the file, which means a // comment could comment content following the #include of the header.